src/Wallabag/CoreBundle/Command/UpdatePicturesPathCommand.php

```diff
@@ -0,0 +1,68 @@
+<?php
+
+namespace Wallabag\CoreBundle\Command;
+
+use Doctrine\ORM\EntityManagerInterface;
+use Symfony\Component\Console\Command\Command;
+use Symfony\Component\Console\Input\InputArgument;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Output\OutputInterface;
+use Symfony\Component\Console\Style\SymfonyStyle;
+use Wallabag\CoreBundle\Repository\EntryRepository;
+
+class UpdatePicturesPathCommand extends Command
+{
+    private EntityManagerInterface $entityManager;
+    private EntryRepository $entryRepository;
+    private string $wallabagUrl;
+
+    public function __construct(EntityManagerInterface $entityManager, EntryRepository $entryRepository, $wallabagUrl)
+    {
+        $this->entityManager = $entityManager;
+        $this->entryRepository = $entryRepository;
+        $this->wallabagUrl = $wallabagUrl;
+        parent::__construct();
+    }
+
+    protected function configure()
+    {
+        $this
+            ->setName('wallabag:update-pictures-path')
+            ->setDescription('Update the path of the pictures for each entry when you changed your wallabag instance URL.')
+            ->addArgument(
+                'old-url',
+                InputArgument::REQUIRED,
+                'URL to replace'
+            );
+    }
+
+    protected function execute(InputInterface $input, OutputInterface $output)
+    {
+        $io = new SymfonyStyle($input, $output);
+
+        $oldUrl = $input->getArgument('old-url');
+
+        $query = $this->entryRepository->createQueryBuilder('e')->getQuery();
+        $io->text('Retrieve existing entries');
+        $i = 1;
+        foreach ($query->toIterable() as $entry) {
+            $content = str_replace($oldUrl, $this->wallabagUrl, $entry->getContent());
+            $entry->setContent($content);
+
+            $previewPicture = str_replace($oldUrl, $this->wallabagUrl, $entry->getPreviewPicture());
+            $entry->setPreviewPicture($previewPicture);
+
+            $this->entityManager->persist($entry);
```

Comment:
not sure the persist is needed, as the entry comes from Doctrine so it's already known by it, but to be checked thanks to the test